Exception("my exception")
def shit():
    raise Exception('shit')
    return "value"